prow/config.yaml

```diff
@@ -1,10 +1,10 @@
 plank:
   default_decoration_config:
     utility_images: # using the tag we identified above
-      clonerefs: "gcr.io/k8s-prow/clonerefs:v20190917-395658c48"
-      initupload: "gcr.io/k8s-prow/initupload:v20190917-395658c48"
-      entrypoint: "gcr.io/k8s-prow/entrypoint:v20190917-395658c48"
-      sidecar: "gcr.io/k8s-prow/sidecar:v20190917-395658c48"
+      clonerefs: "morvencao/k8s-prow-clonerefs:v20190917-395658c48"
```

Comment:
why use your own repo? Can we use quay.io?

Reply:
These are multi-arch images, currently `quay.io` doesn't support multi-arch image without suffix, so we use docker.io temporarily.

Reply:
Ah, I see, thanks.